app/resources/messages.py

```diff
@@ -97,18 +97,58 @@ def _message_save(self, message, is_draft, draft_id):
 
         if is_draft is True:
             Modifier().del_draft(draft_id)
-        return MessageSend._alert_recipients(message.data.msg_id, message.data.thread_id)
 
-    @staticmethod
-    def _alert_recipients(msg_id, thread_id):
-        """used to alert user once messages have been saved"""
-        recipient_email = settings.NOTIFICATION_DEV_EMAIL  # TODO change this when know more about party service
-        alert_user = AlertUser()
-        alert_status, alert_detail = alert_user.send(recipient_email, msg_id)
-        resp = jsonify({'status': '{0}'.format(alert_detail), 'msg_id': msg_id, 'thread_id': thread_id})
-        resp.status_code = alert_status
+        # listener errors are logged but still a 201 reported
+        MessageSend._alert_listeners(message.data)
+
+        resp = jsonify({'status': '201', 'msg_id': message.data.msg_id, 'thread_id': message.data.thread_id})
+        resp.status_code = 201
         return resp
 
+    @staticmethod
+    def _alert_listeners(message):
```

Comment:
Yeah I just followed through the code and you're right. 
I was getting a strange thing happening a few days ago where I was getting an error log telling me that gov.notify had reached max number of messages it could send even when I was just sending from frontend. Not sure what could have been triggering that